parlai/mturk/webapp/dev/app.jsx

```diff
@@ -758,7 +760,7 @@ class AssignmentFeedback extends React.Component {
     var bsStyle = null;
     let given_feedback = null;
     let received_feedback = null;
-    if (review_data !== undefined) {
+    if (!!review_data) {
```

Comment:
TIL `!!` converts to `true`

Reply:
Right, it's a truthiness check. Technically I could just leave it as `if (review_data) {` but I like to make the check explicit.